flume/doc/quick_start_guide.md

```diff
@@ -1 +1,154 @@
+This quick start overviews the steps a newbie programmer will have to follow in order to get familiar with Cygnus and its basic functionality. For more detailed information, please refer to the [Cygnus GitHub](https://github.com/telefonicaid/fiware-connectors).
+
+##Basic knowlegde about Cygnus
+Previous to installing and using it, you should know a couple of things about Cygnus.
+
+###What is is for?
+Cygnus is a connector in charge of persisting Orion context data in certain configured third-party storages, creating an historical view of such data. In other words, Orion only stores the last value regarding an entity's attribute, and if an older value is required then you will have to persist it in other storage, value by value, using Cygnus.
+
+###How does it receives context data from Orion Context Broker?
+Cygnus uses the subscription/notification feature of Orion. A subscription is made in Orion on behalf of Cygnus, detailing which entities we want to be notified for when an update occurs on any of those entities attributes.
+
+###Which storages is it able to integrate?
+Current stable release (0.7.1) is able to persist Orion context data in:
+
+* HDFS, the Hadoop distributed file system.
+* MySQL, the well-know relational database manager.
+* CKAN, an Open Data platform.
+
+###Which is its basic architecture?
+Internally, Cygnus is based on Apache Flume. In fact, Cygnus is a Flume agent, which is basically composed of a <i>source</i> in charge of receiving the data, a <i>channel</i> where the source puts the data once it has been transformed into a Flume event, and a <i>sink</i>, which takes Flume events from the channel in order to persist the data within its body into a third-party storage.
+
+As said, Cygnus is able to persist in HDFS, MySQL and CKAN, thus there exists a sink for each one of those storages.
+
+##Installing Cygnus
+Open a terminal and simply configure the FIWARE repository if not yet configured and use your applications manager in order to install the latest version of Cygnus (CentOS/RedHat example):
+
+```
+$ cat > /etc/yum.repos.d/fiware.repo <<EOL
+[Fiware]
+name=FIWARE repository
+baseurl=http://repositories.testbed.fi-ware.eu/repo/rpm/x86_64/
+gpgcheck=0
+enabled=1
+EOL
+$ yum install cygnus
+```
+
+When installing through the RPM, Cygnus is started as a service. Stop it since this Quick Start Guide will run Cygnus from the command line.
+
+```
+$ service cygnus stop
+```
+
+##Configuring a test agent
+This kind of agent is the simplest one you can configure with Cygnus. It is based on a standard `HTTPSource`, a `MemoryChannel` and a `OrionTestSink`. Don't worry about the configuration details, specially those about the source; simply think on a Http listener waiting for Orion notifications on port TCP/5050 and sending that notifications in the form el Flume events to a testing purpose sink that will not really persist anything in a third-party storage, but will log the notified context data.
```

Comment:
All fixed in https://github.com/telefonicaid/fiware-connectors/commit/87780244e689ba2772acee7da4bfb2fd8bd5263a